src/OSPSuite.Core/Domain/ObjectPathKeywords.cs

```diff
@@ -62,11 +62,28 @@ public static class ObjectPathKeywords
       public static readonly string ALL_FLOATING_MOLECULES = addKeyword("ALL_FLOATING_MOLECULES");
 
       /// <summary>
-      /// Represents the keywords allowing us to identify a neighborhood between two containers
-      /// For example CONTAINER1|/<NBH/>CONTAINER2/<NBH/> means the neighborhood between CONTAINER1 and CONTAINER2
+      ///    Represents the keywords allowing us to identify a neighborhood between two containers
+      ///    For example CONTAINER1|/<NBH />CONTAINER2/<NBH /> means the neighborhood between CONTAINER1 and CONTAINER2
       /// </summary>
       public static readonly string NBH = addKeyword("<NBH>");
 
+      /// <summary>
+      ///    Search for a compartment with the same name as the last part preceding the keyword under Lumen
+      /// </summary>
+      /// <example>
+      ///   Absolute Path:
+      ///   <code>
+      ///      Organism|SmallIntestine|Mucosa|Duodenum|LUMEN_SEGMENT => Organism|Lumen|Duodenum
+      ///      Organism|SmallIntestine|Duodenum|LUMEN_SEGMENT|Volume  => Organism|Lumen|Duodenum|Volume
+      ///      Organism|SmallIntestine|Duodenum|LUMEN_SEGMENT|Intracellular => Organism|Lumen|Duodenum|Intracellular
```

Comment:
The examples for the absolute path above are not wrong, however using _absolute paths_ with the LUMEN_SEGMENT keyword does not make much sense from the modeling point of view.
So I would completely remove these examples from the comment.

Reply:
Sure. I copied them from the GitHub issue. But I agree